R/ioslides_presentation.R

```diff
@@ -289,9 +293,17 @@ ioslides_presentation <- function(number_sections = FALSE,
     template <- pkg_file("rmd/ioslides/default.html")
   args <- c(args, "--template", pandoc_path_arg(template))
 
-  # html dependency for ioslides
-  extra_dependencies <- append(extra_dependencies,
-                               list(html_dependency_ioslides()))
+  # Add ioslides HTML dependencies
+  # If bslib is relevant, we want to register a _deferred_ version of the CSS dependency so
+  # it can update in response to Shiny's session$setCurrentTheme() (e.g., bslib::bs_themer())
+  # https://rstudio.github.io/bslib/articles/theming.html#dynamic-theming-in-shiny
+  theme <- resolve_theme(theme)
```

Comment:
> Using `ioslides_presentation: theme: cerulean` will issue no warning or error. It will do nothing, so not harmful but some users could expect this to work as other formats.

This _does_ do something though -- it brings in Bootstrap! So, I don't think we want to introduce an error in this case -- I'm sure folks are relying on this functionality.